demo/index.html

```diff
@@ -1,81 +1,758 @@
+                            fill="none"
+                            stroke="currentColor"
+                            stroke-width="2"
+                            stroke-linecap="round"
+                            stroke-linejoin="round"
+                            class="icon icon-tabler icons-tabler-outline icon-tabler-external-link"
+                        >
+                            <path stroke="none" d="M0 0h24v24H0z" fill="none" />
+                            <path
+                                d="M12 6h-6a2 2 0 0 0 -2 2v10a2 2 0 0 0 2 2h10a2 2 0 0 0 2 -2v-6"
+                            />
+                            <path d="M11 13l9 -9" />
+                            <path d="M15 4h5v5" />
+                        </svg>
+                    </a>
+                </aside>
+                <main>
+                    <section id="introduction">
+                        <h1><a href="#introduction">Drag and Drop Touch</a></h1>
+                        <p>
+                            A polyfill that enables HTML5 drag drop support on mobile (touch)
+                            devices.
+                        </p>
+                        <p>
+                            See the Github page
+                            <a
+                                href="https://github.com/drag-drop-touch-js/dragdroptouch"
+                                target="_blank"
+                                >here</a
+                            >.
+                        </p>
+                    </section>
+                    <section id="why">
+                        <h2>
+                            <a href="#why">Why</a>
+                        </h2>
+                        <p>
+                            The HTML5 specification includes support for drag and drop operations.
+                        </p>
+                        <p>
+                            Unfortunately, this specification is based on mouse events, rather than
+                            pointer events, and so most mobile browsers do not implement it.
+                        </p>
+                        <p>
+                            As such, applications that rely on HTML5 drag and drop have reduced
+                            functionality when running on mobile devices.
+                        </p>
+                        <p>
+                            The DragDropTouch class is a polyfill that translates touch events into
+                            standard HTML5 drag drop events.
+                        </p>
+                        <p>
+                            If you add the polyfill to your pages, drag and drop operations should
+                            work on mobile devices just like they do on the desktop.
+                        </p>
+                    </section>
+                    <section id="installation">
+                        <h2><a href="#installation">Installation</a></h2>
+                        <div>
```

Comment:
In order to keep things semantically marked up, this should be a `<section>`, not a `<div>` (if something has its own header tag, that's a section =)